### DISCUSSION TITLE
[component] 缩略轴 slider 的 handler 支持图标 icon 配置

### DISCUSSION THREAD

**Discussion by bigbigqiang** [antvis/G2Plot#2349] [Ideas]
* **G2Plot Version**:
* **Platform**:
* **Mini Showcase(like screenshots)**:
* **CodePen Link**:

**Comment by visiky** [comment]
ref: https://github.com/antvis/component/pull/220

**checklist**
- [x] handlerStyle fill、highlightFill 设置不生效
- [ ] 增加图标设置
- [x] api 文档修改 #2347

**Comment by visiky** [comment]
暂不支持 slider 配置 handler 图标，后续再支持
